dds_web/database/models.py

```diff
@@ -127,6 +129,9 @@ class User(db.Model):
     # One user can have many email addresses
     emails = db.relationship("Email", backref="users", lazy="dynamic", cascade="all, delete-orphan")
 
+    # One user can create many projects
+    created_projects = db.relationship("Project", backref="user", cascade="all, delete-orphan")
```

Comment:
The others have "users" because one row in those tables can have many associated users. In this case a project can only be created by one user, so it should be `user` 👍🏻

Reply:
I see. Thanks for clarification.

Reply:
Any time!